### PR TITLE
Enable pylint diagnostics `too-many-statements`

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -2,7 +2,6 @@
 disable=bad-continuation,         ######## Disabled because of conflicts with black ########
         consider-using-enumerate, ######## Temporary disabled ########
         import-error,
-        too-many-statements,
```
Comment (nedrebo) [inline .pylintrc:5]: This looks like the _enabling_ of a diagnostics.

**Review by nedrebo — CHANGES_REQUESTED** [review]

File: .pylintrc (hunk 1 above)
Comment (nedrebo) [reply .pylintrc:5]: The commit message should also be more verbose, not many know what `too-many-statements` is.

Proposal:
```
Enabling pylint diagnostics `too-many-statements`
```

Comment (SatjaSivcev) [reply .pylintrc:5]: done, but it is 'Dissabling'

Comment (eskaur) [reply .pylintrc:5]: I'm still confused. You are removing a disable-statement, and thus enabling it, no?

Comment (SatjaSivcev) [reply .pylintrc:5]: My mistake.

Comment (SatjaSivcev) [reply .pylintrc:5]: Well, the problem was that this enables diagnostics, but disables too many statements :)

Comment (SatjaSivcev) [reply .pylintrc:5]: I've renamed the commit message.

Comment (nedrebo) [reply .pylintrc:5]: Looks good now.

Comment (nedrebo) [reply .pylintrc:5]: PR title is still weird.

Comment (SatjaSivcev) [reply .pylintrc:5]: renamed it to: Enable pylint diagnostics `too-many-statements`

Comment (SatjaSivcev) [reply .pylintrc:5]: Need your approval @nedrebo  to merge.

**Review by nedrebo — APPROVED** [review]

**Pull request merged**